cppcache/benchmark/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set(THREADS_PREFER_PTHREAD_FLAG TRUE)
+
+find_package(Threads REQUIRED)
```

Comment:
Revert these changes since they are now correctly in dependencies/benchmark.

Reply:
Done